webdriver/tests/bidi/network/conftest.py

```diff
@@ -11,25 +11,30 @@
 
 
 @pytest.fixture
-def fetch(bidi_session, top_context):
+def fetch(bidi_session, top_context, configuration):
     """Perform a fetch from the page of the provided context, default to the
     top context.
     """
-    async def fetch(url, method="GET", headers=None, context=top_context):
+    async def fetch(url, method="GET", headers=None, context=top_context, timeout=1000):
```

Comment:
Just wondering, since in python the default unit for time is rather seconds I think, it might be confusing to have this timeout in ms? Should we use seconds and ` * 1000` when sending that over to `script.evaluate`

Reply:
Fair point. We do something similar in the [webdriver classic client](https://github.com/web-platform-tests/wpt/blob/f69cc2c952a97e745446a6026559292a96340fd8/tools/webdriver/webdriver/client.py#L42).

Reply:
Renamed to `timeout_in_seconds` to avoid confusion